symbolic-base/src/ZkFold/Base/Algebra/Basic/Class.hs

```diff
@@ -475,7 +475,8 @@ instance AdditiveMonoid Natural where
 instance Semiring Natural
 
 instance SemiEuclidean Natural where
-    divMod = Haskell.divMod
+    divMod m 0 = (0, m)
```

Comment:
Is this specific behaviour required somewhere?

Reply:
Yeah, division by zero happens when a test for Symbolic `divMod` runs
Now that I think about it, maybe it is better to modify the test so that it doesn't choose zero as a denominator